include/deal.II/lac/la_vector.templates.h

```diff
@@ -592,15 +592,14 @@ namespace LinearAlgebra
 
     // Other version of the following
     //  out << size() << std::endl << '[';
-    // Reason: operator<< seems to use some resources  that lead to problems in
-    // a multithreaded environment.
-    const size_type sz = this->size();
-    char            buf[16];
-#ifdef DEAL_II_WITH_64BIT_INDICES
+    // Reason: operator<< seems to use some resources that lead to problems in
+    // a multithreaded environment. We convert the size index to unsigned long
+    // long int that is at least 64 bits to be able to output it on all
```

Comment:
Nit-picking: It's probably easier to read if we don't break the line for `unsigned long long int`. Same below.

Reply:
Fixed!